src/huggingface_hub/commands/user.py

```diff
@@ -196,7 +204,14 @@ def run(self):
             print("Not logged in")
             exit()
         HfFolder.delete_token()
-        self._api.logout(token)
+        try:
+            self._api.logout(token)
+        except HTTPError as e:
+            if e.response.status_code == 400:
+                # Logging out with an access token will return a client error.
+                HfApi.unset_access_token()
```

Comment:
I think we want to call this even if there is no exception since it's always set by `_login`